src/physics/src/boussinesq_buoyancy.C

```diff
@@ -37,13 +37,6 @@
 
 namespace GRINS
 {
-
-  BoussinesqBuoyancy::BoussinesqBuoyancy( const std::string& physics_name, const GetPot& input )
-    : BoussinesqBuoyancyBase(physics_name,input)
```

Comment:
This is great.  I'd forgotten how annoying writing some of these shims was.  And I don't think all the annoyance was from GRINS; I need to go through the libMesh sources to look for opportunities to use inheriting constructors there too.  I didn't even think to do that when I was skimming our sources looking for good spots to use delegating constructors.